src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriverExtension.java

```diff
@@ -51,6 +51,15 @@ public final String id() {
 		return NAME;
 	}
 
+	private final String uriSchema()  {
```

Comment:
`SCHEMA_PROVIDER` is a configuration validation schema. This doesn't relate to the URI schema. I don't think that this method is needed